board/app/controllers/thread_controller.php

```diff
@@ -0,0 +1,108 @@
+<?php
+class ThreadController extends AppController {
+    
+/*Create a new comment (Thread)
+* :: - STATIC FUNCTION, can be called from the class name
+* -> - INSTANCE, can only be called from an instance of the class.
+*/
+public function create(){
+
+        $thread = new Thread();
+        $comment = new Comment();
+        $page = Param::get('page_next', 'create');
+        
+        switch ($page) {
+           case 'create':
+           break;
+          
+           case 'create_end':
+            $thread->title = Param::get('title');
+            $comment->username = Param::get('username');
+            $comment->body = Param::get('body');
+ 
+           try {
+               $thread->create($comment);
+               } catch (ValidationException $e) {
+               $page = 'create';
+            }
+           break;
+           
+           default:
+            throw new NotFoundException("{$page} is not found");
+            break;
+         }  
+
+       $this->set(get_defined_vars());
+       $this->render($page);
+    }
+ 
+
+//Instance of the Thread model, and its static functions getAll()
```

Comment:
noted po. thank you :bow:
changed the comment po to:   //Displays all the threads